app/src/main/java/com/example/lunchtray/LunchTrayScreen.kt

```diff
@@ -189,8 +194,9 @@ fun LunchTrayApp() {
                         navController.popBackStack(LunchTrayScreen.Start.name, inclusive = false)
                     },
                     modifier = Modifier
-                        .padding(dimensionResource(R.dimen.padding_medium))
                         .verticalScroll(rememberScrollState())
+                        .padding(innerPadding)
```

Comment:
Combine the two `.padding()`  function calls into 1 or remove 1 depending on what you need.